sts3x/Makefile

```diff
@@ -1,43 +1,19 @@
-sensirion_common_dir := ../embedded-common
-sts_common_dir := ../sts-common
-sw_i2c_dir := ${sensirion_common_dir}/sw_i2c
-hw_i2c_dir := ${sensirion_common_dir}/hw_i2c
+UNAME_S := $(shell uname -s)
```

Comment:
This may annoying when trying to cross compile.

Reply:
true, didn't think about that. What if we completely scrap it and revert back to the previous behavior of only building the stubs (though circle CI would still build the linux one)?

Reply:
Sounds good.